cluster-autoscaler/simulator/nodes.go

```diff
@@ -20,31 +20,28 @@ import (
 	kube_api "k8s.io/kubernetes/pkg/api"
 	kube_client "k8s.io/kubernetes/pkg/client/unversioned"
 	"k8s.io/kubernetes/pkg/fields"
-	cmd "k8s.io/kubernetes/pkg/kubectl/cmd"
 	"k8s.io/kubernetes/plugin/pkg/scheduler/schedulercache"
 )
 
 // GetRequiredPodsForNode returns a list od pods that would appear on the node if the
-// node was just created (like deamonset and manifest-run pods). It reuses kubectl
-// drain command to get the list.
+// node was just created (like deamonset and manifest-run pods).
 func GetRequiredPodsForNode(nodename string, client *kube_client.Client) ([]*kube_api.Pod, error) {
-	podsToRemoveList, _, _, err := cmd.GetPodsForDeletionOnNodeDrain(client, nodename,
-		kube_api.Codecs.UniversalDecoder(), true, true)
+	allPodList, err := client.Pods(kube_api.NamespaceAll).List(
```

Comment:
@mwielgus after #26667 there's no `cmd.GetPodsForDeletionOnNodeDrain`.
Please advice how to procede